extensions/wikia/ArticleVideo/scripts/article-video.js

```diff
@@ -0,0 +1,103 @@
+require(['wikia.window', 'wikia.onScroll', 'ooyalaVideo'], function (window, onScroll, OoyalaVideo) {
+	$(function () {
+		var $video = $('#article-video'),
+			$videoContainer = $video.find('.video-container'),
+			$videoThumbnail = $videoContainer.find('.video-thumbnail'),
+			$closeBtn = $videoContainer.find('.close'),
+			ooyalaVideo,
```

Comment:
`OoyalaVideo` and `ooyalaVideo` in the same method? it's very likely someone will make a typo here ;)